index.html

```diff
@@ -652,231 +664,290 @@ <h3>
           and {{PositionOptions/maximumAge}}.
         </p>
         <pre class="idl">
-  dictionary PositionOptions {
-    boolean enableHighAccuracy = false;
-    [Clamp] unsigned long timeout = 0xFFFFFFFF;
-    [Clamp] unsigned long maximumAge = 0;
-  };
-
-  </pre>
-        <p>
-          In ECMAScript, the {{PositionOptions/enableHighAccuracy}},
```

Comment:
Reading this section that is dropped, it just appeared to me the algorithmic prose does not really account for the cases when the timeout or maximumAge members are not present.

For enableHighAccuracy, the spec at least tries to cater for that case (weakly worded, but still):

>possibly taking into account the value of the enableHighAccuracy variable

Do others see an issue here?

Reply:
> the cases when the timeout or maximumAge members are not present.

ah! because they have a default value, so they are always present (i.e., in C++ or whatever, they are always initialized with something). 

>possibly taking into account the value of the enableHighAccuracy variable
> Do others see an issue here?

The "possibly taking into account the value of the enableHighAccuracy variable" is more around the accuracy of the returned position, not the value of `enableHighAccuracy`.